x-pack/test/functional/apps/timelion/feature_controls/timelion_spaces.ts

```diff
@@ -23,29 +24,44 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       before(async () => {
         // we need to load the following in every situation as deleting
         // a space deletes all of the associated saved objects
-        await esArchiver.load('x-pack/test/functional/es_archives/timelion/feature_controls');
+        // await esArchiver.load('x-pack/test/functional/es_archives/timelion/feature_controls');
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/timelion.json'
+        );
+
         await spacesService.create({
           id: 'custom_space',
           name: 'custom_space',
           disabledFeatures: [],
         });
+
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/timelion_custom_space.json',
```

Comment:
What is unique about this one? The space isn't defined in the archive, it's only set on import.

Reply:
The previous archive had three saved objects in one space and a single saved object in another space. In order to reproduce what the previous test was doing we split the archive into two kbnArchives, one with the intention of going into the custom space and only containing the single timelion sheet. I think ultimately there isn't a good justification for this that we know of, but we were basically operating under the assumption that the original test author did what they did for a good reason.

Reply:
Works for me. Timelion is getting the axe in 7.15, so I am not worried.